prusti-contracts/prusti-specs/src/rewriter.rs

```diff
@@ -110,12 +112,17 @@ impl AstRewriter {
         //   of a single identifier; without the double negation, the `Return`
         //   terminator in MIR has a span set to the one character just after
         //   the identifier
+        let (return_type, return_modifier) = if spec_type == SpecItemType::Termination {
+            (quote! {Int}, quote! {Int::new(0) + })
```

Comment:
Could you try using `quote_spanned!` here to see whether that fixes the `test_failing_crate` test?